pkgs/top-level/all-packages.nix

```diff
@@ -34603,9 +34603,7 @@ with pkgs;
     enableDbusUi = false;
   };
 
-  parsec-bin = callPackage ../applications/misc/parsec/bin.nix {
-    ffmpeg = ffmpeg_4;
```

Comment:
It may make sense to just update this override to `ffmpeg_5`, since the binary does support multiple ffmpeg versions and doesn't seem to want 5 explicitly.

Reply:
I was encouraged to remove options from the all-packages.nix file to reduce clutter in it.
Is there any reason why one would want to stick on older ffmpeg_3/ffmpeg_4?

Reply:
Yes, let's get implementation details out of all-packages. This also aids with https://github.com/NixOS/rfcs/pull/140.